### DISCUSSION TITLE
Support for opening JPEG-XL files

### DISCUSSION THREAD

**Discussion by bakkeby** [bakkeby/sxiv-flexipatch#4] [Ideas]
Support for JPEG-XL (.jxl) files can be added to Imlib2 which allows for sxiv to open and display such files.

Refer to this repository:
   - https://github.com/alistair7/imlib2-jxl

**Comment by N-R-K** [comment]
I found a couple other custom imlib2 loaders. 

https://github.com/arp242/imlib2-heif
https://github.com/vi/imlib2-avif
https://github.com/vi/imlib2-bpg

I did not test any of them, so I have no clue if/or how well they work.


Also worth noting, it seems that imlib2 native HEIF support is WIP. https://phab.enlightenment.org/D12267
